src/main/java/org/cascadebot/cascadebot/permissions/CascadePermission.java

```diff
@@ -27,7 +27,7 @@
     private final Module module;
 
 
-    private CascadePermission(String permissionRaw, boolean defaultPerm, Module module, Permission... discordPerm) {
+    public CascadePermission(String permissionRaw, boolean defaultPerm, Module module, Permission... discordPerm) {
```

Comment:
Why has this been changed? There are factory methods to create a CascadePermission